sdcm/sct_events/database.py

```diff
@@ -362,10 +362,19 @@ def __init__(self, node: str, severity=Severity.NORMAL, **__):
         super().__init__(node=node, severity=severity)
 
 
+class ScyllaSysconfigSetupEvent(ScyllaDatabaseContinuousEvent):
```

Comment:
did you remove publishing this event? If so, lets remove it.

Reply:
Nope, just missed the `publish_event` argument. I added it now.